code/api/src/config/params.json

```diff
@@ -25,6 +25,44 @@
         "id": 2,
         "name": "Female"
       }
+    },
+    "style": {
+      "classic": {
+        "id": 1,
+        "name": "Classic"
+      },
+      "modern": {
+        "id": 2,
+        "name": "Modern"
+      },
+      "formal": {
+        "id": 3,
+        "name": "Formal"
+      },
+      "casual": {
+        "id": 4,
+        "name": "Casual"
+      },
+      "adaptable": {
+        "id": 5,
+        "name": "Adaptable"
+      },
+      "formal_classic": {
+        "id": 6,
+        "name": "Formal but Classic"
+      },
+      "formal_modern": {
+        "id": 7,
+        "name": "Formal but Modern"
+      },
+      "casual_classic": {
+        "id": 8,
+        "name": "Casual but Classic"
+      },
+      "casual_modern": {
+        "id": 9,
+        "name": "Casual but Modern"
+      }
```

Comment:
This looks good. But I'm wondering if this can be done the same way user roles were done. User roles look simpler but it looks like the enum might come from the id?

Reply:
Also, should we add an enum for unfinished or not yet started?

Reply:
I agree here. Enums might work better. Also let's get back together and make final map/graph with all the choices we will have.

Reply:
We can drop the combined styles. We'll have to add something like "New User" as well as "Survey Incomplete".